frontend/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "yohoard-ui",
```

Comment:
Ale ten "yohoard-ui" to mógłby zostać :P Po co wracać do generycznej nazwy "frontend"?